bigquery_etl/shredder/delete.py

```diff
@@ -396,9 +401,15 @@ def main():
             )
     with ThreadPool(args.parallelism) as pool:
         glean_targets = find_glean_targets(pool, client)
+    with ThreadPool(args.parallelism) as pool:
+        experiment_analysis_targets = find_experiment_analysis_targets(pool, client)
```

Comment:
there's no need to recreate this thread pool
```suggestion
    with ThreadPool(args.parallelism) as pool:
        glean_targets = find_glean_targets(pool, client)
        experiment_analysis_targets = find_experiment_analysis_targets(pool, client)
```